src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
```

Comment:
remove commented code

Reply:
[TEST REPORT LINK](https://07Spirit.github.io/layout_search-bar-airbnb/report/html_report/)

I dont know why, but my test link in any programm dont open.